src/utils/file-path.ts

```diff
@@ -0,0 +1,56 @@
+import * as R from 'ramda'
+import type {RuleParamsBase} from './types'
+
+/**
+ * For all created/modified files finds those starting with includePaths (excluding those starting with excludePaths).
+ *
+ * @param danger Danger instance
+ * @param excludePaths paths to exclude
+ * @param includePaths paths to include
+ */
+export const filterPaths = (params: Omit<RuleParamsBase, 'fail'>): Array<string> => {
```

Comment:
@quesabe please split the exported entities into different files. One export per file.